src/main/java/de/tum/in/www1/artemis/service/ModelingSubmissionService.java

```diff
@@ -206,6 +209,7 @@ public ModelingSubmission save(ModelingSubmission modelingSubmission, ModelingEx
         modelingSubmission.setType(SubmissionType.MANUAL);
         modelingSubmission.setParticipation(participation);
         modelingSubmission = modelingSubmissionRepository.save(modelingSubmission);
+        submissionVersionService.save(modelingSubmission, username);
```

Comment:
can we wrap this in a generic try {... } catch(Exception ex) {...} so that it never can happen that any kind of (runtime) exception (we did not think of) in the submission version service makes problem with saving the normal submission? (which should really always work!)

Reply:
Yes, makes sense. I'll add a try catch block.